tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2045,6 +2053,10 @@ Status ConvertConv2DHelper(const OpConverterParams* params, int group,
         static_cast<const float*>(const_weights_rsck.values);
     std::copy_n(weights_ptr, const_weights_rsck.count,
                 weights_rsck.GetPointer<float>());
+    // Tensorrt accepts the filter in KCRS mode, while the tensorflow tensor is
+    // set up in RSCK.
+    TF_RETURN_IF_ERROR(params->converter->TransposeTensor(
+        filter_tensor, {3, 2, 0, 1}, &filter_tensor, node_def, "to_KCRS"));
```

Comment:
We must move the transpose **before** adding the TensorRT convolution layer, otherwise building the engine will fail with shape mismatch on the second input.